Lib/multiprocessing/dummy/__init__.py

```diff
@@ -33,7 +33,9 @@
 
 class DummyProcess(threading.Thread):
 
-    def __init__(self, group=None, target=None, name=None, args=(), kwargs={}):
+    def __init__(self, group=None, target=None, name=None, args=(), kwargs=None):
+        if kwargs is None:
```

Comment:
we don't need to do this. threading.Thread already accepts None and does this itself.